templates/prometheus.go

```diff
@@ -34,42 +35,48 @@ var resourceSelector = metav1.LabelSelector{
 	},
 }
 
+const (
+	//TODO: Construct monitoring dns name dynamically based on AWS and IBM Cloud Monitoring regions
+	MonitoringDNSName = "ingest.us-south.monitoring.cloud.ibm.com"
+)
+
 var (
 	KubeRBACProxyPortNumber             int    = 9339
 	PrometheusServingCertSecretName     string = "prometheus-serving-cert-secret"
 	PrometheusKubeRBACPoxyConfigMapName string = "prometheus-kube-rbac-proxy-config"
+	MonitoringURL                       string = fmt.Sprintf("https://%s/prometheus/remote/write", MonitoringDNSName)
 )
 
-// var metrics = []string{
-// 	"job:ceph_versions_running:count",
-// 	"job:ceph_pools_iops_bytes:total",
-// 	"job:ceph_pools_iops:total",
-// 	"job:kube_pv:count",
-// 	"job:ceph_osd_metadata:count",
-// 	"ceph_health_status",
-// 	"ceph_cluster_total_used_raw_bytes",
-// 	"ceph_cluster_total_bytes",
-// 	"cluster:kubelet_volume_stats_used_bytes:provisioner:sum",
-// 	"cluster:kube_persistentvolumeclaim_resource_requests_storage_bytes:provisioner:sum",
-// }
+var metrics = []string{
+	"job:ceph_versions_running:count",
+	"job:ceph_pools_iops_bytes:total",
+	"job:ceph_pools_iops:total",
+	"job:kube_pv:count",
+	"job:ceph_osd_metadata:count",
+	"ceph_health_status",
+	"ceph_cluster_total_used_raw_bytes",
+	"ceph_cluster_total_bytes",
+	"cluster:kubelet_volume_stats_used_bytes:provisioner:sum",
+	"cluster:kube_persistentvolumeclaim_resource_requests_storage_bytes:provisioner:sum",
+}
```

Comment:
These metrics are specific to DF and DFC offering so they should come from datafoundation or datafoundationclient package

Reply:
Moved ceph related metrics and alerts to `datafoundation/templates/storagecluster.go`